service/src/main/java/org/apache/celeborn/server/common/service/config/FsConfigServiceImpl.java

```diff
@@ -51,28 +53,36 @@ public synchronized void refreshAllCache() {
 
     SystemConfig systemConfig = null;
     Map<String, TenantConfig> tenantConfs = new HashMap<>();
+    Map<UserIdentifier, TenantConfig> tenantUserConfs = new HashMap<>();
     try (FileInputStream fileInputStream = new FileInputStream(configurationFile)) {
       Yaml yaml = new Yaml();
       List<Map<String, Object>> dynamicConfigs = yaml.load(fileInputStream);
       for (Map<String, Object> settings : dynamicConfigs) {
         String tenantId = (String) settings.get(CONF_TENANT_ID);
+        String userId = (String) settings.get(CONF_TENANT_USER_ID);
         String level = (String) settings.get(CONF_LEVEL);
         Map<String, String> config =
             ((Map<String, Object>) settings.get(CONF_CONFIG))
                 .entrySet().stream()
                     .collect(Collectors.toMap(Map.Entry::getKey, a -> a.getValue().toString()));
-        if (ConfigLevel.TENANT.name().equals(level)) {
-          TenantConfig tenantConfig = new TenantConfig(this, tenantId, null, config);
-          tenantConfs.put(tenantId, tenantConfig);
+        if (ConfigLevel.TENANT_USER.name().equals(level)) {
+          TenantConfig tenantConfig = new TenantConfig(this, tenantId, userId, config);
+          tenantUserConfs.put(new UserIdentifier(tenantId, userId), tenantConfig);
```

Comment:
Does the `tenantUserConfs` merge with `tenantConfs` given same tenant id? For example,
```
-  tenantId: tenant_id1
   level: TENANT
   config:
     celeborn.client.push.buffer.initial.size: 10k
     celeborn.client.push.buffer.initial.size.only: 100k
     celeborn.worker.fetch.heartbeat.enabled: false
     celeborn.test.tenant.timeoutMs.only: 100s
     celeborn.test.tenant.enabled.only: false
     celeborn.test.tenant.int.only: 10
     celeborn.client.push.queue.capacity: 1024

-  tenantId: tenant_id1
   level: TENANT_USER
   name: AngersZhuuu
   config:
     celeborn.client.push.buffer.initial.size: 20k
     celeborn.client.push.buffer.initial.size.only: 100k
     celeborn.worker.fetch.heartbeat.enabled: false
```
What's the value of `celeborn.client.push.queue.capacity` for user `AngersZhuuu`?

Reply:
1024